scripts/core/CONTRIB.rst

```diff
@@ -1,71 +1,177 @@
-
 <%
     OneApi=tags['$OneApi']
     x=tags['$x']
     X=x.upper()
 %>
-===================
+====================
  Contribution Guide
-===================
+====================
 
-This contribution guide explains how to contribute to the Unified Runtime 
-project and what processes you must follow in order to have your changes 
+This contribution guide explains how to contribute to the Unified Runtime
+project and what processes you must follow in order to have your changes
 accepted into the project.
 
+.. important::
+
+    Before making a contribution you *should* determine if the change should be
+    made directly to the core specification or introduced as an experimental
+    feature. The criteria we use to make this distinction are as follows:
+
+    *   The feature exists to enable an experimental feature in a parallel
+        language runtime being built on top of Unified Runtime.
+
+    *   The design phase of the feature is expected to span multiple oneAPI
+        releases.
+
+    *   A proof of concept implementation exists for a single adapter but
+        multiple adapters are intended to be supported.
```

Comment:
```suggestion
        multiple adapters are intended to be supported. If you are unsure whether the feature can be supported by certain adapters then seek the advise of an appropriate stakeholder. It is important to consider as early as possible whether the feature is appropriate for other adapters and seek the input of relevant stakeholders for each adapter in order to ensure a portable implementation.
```

Reply:
I have added a modified version of this.